### DISCUSSION TITLE
Raindrop forecasting?

### DISCUSSION THREAD

**Discussion by JauJieYou** [WenjieDu/PyPOTS#400] [General]
Dear Sir
  Is it possible to offer Raindrop forecasting, not only classification?
Thanks

**Comment by WenjieDu** [comment]
Hi there 👋,

Thank you so much for your attention to PyPOTS! You can follow me on GitHub to receive the latest news of PyPOTS. If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

I have received your message and will respond ASAP. Thank you for your patience! 😃

Best,
Wenjie

**Comment by WenjieDu** [comment]
@JauJieYou Yes, I think so, though the Raindrop model is originally proposed for classification.

This feature will be added to Raindrop in future updates. To receive the latest news, you should follow me on GitHub and star 🌟  [PyPOTS repo](https://github.com/WenjieDu/PyPOTS).
